src/webui/webapplication.cpp

```diff
@@ -735,6 +737,28 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hashes" << "location");
+
+    QStringList hashes = request().posts["hashes"].split("|");
+    QString newLocation = request().posts["location"].trimmed();
+
+    // check if location valid
+    if(newLocation.isEmpty() || !QDir(newLocation).exists()) return;
+
+    foreach (const QString &hash, hashes) {
+        BitTorrent::TorrentHandle *const torrent = BitTorrent::Session::instance()->findTorrent(hash);
+        if (torrent) {
```

Comment:
maybe saying explicitly that torrent != null ?

Reply:
Shall I change this :P ? Just wanted to ensure a consistent style across the code, check the [action_command_recheck](https://github.com/Flipkart/qBittorrent/blob/shatabdi/setLocation/src/webui/webapplication.cpp#L762) func.